src/sentry/utils/query.py

```diff
@@ -317,4 +311,11 @@ def bulk_delete_objects(model, limit=10000, transaction_id=None, logger=None, **
 
     cursor = connection.cursor()
     cursor.execute(query, params)
+
+    if logger is not None:
+        logger.info('object.delete.bulk_executed', extra={filters.items() + [
```

Comment:
This definitely isn't what you intended here. Pretty sure this is going to raise a `TypeError` from trying to convert to a `set`.